test/HarpoonKrakenTests.cpp

```diff
@@ -4,6 +4,62 @@
 #include "MockKraken.h"
 #include "Harpoon.h"
 #include "Death.h"
+#include <future>
+#include <atomic>
+
+void* HarpoonKrakenTests::SendHello(void* arg) {
+   std::string address = *(reinterpret_cast<std::string*>(arg));
+   Kraken server;
+   server.SetLocation(address);
+   server.MaxWaitInMs(1000); // 1 second
+
+   uint8_t h = 'h';
+   uint8_t e = 'e';
+   uint8_t l = 'l';
+   uint8_t o = 'o';
+   uint8_t end = '\0';   
+   std::vector<uint8_t> hello{h,e,l,l,o,end};
+
+   for (auto c : hello) {
+      auto status = server.SendTidalWave({c});
+      LOG(INFO) << "SendTidalWave returned: " << static_cast<int> (status);
+   }
+   server.FinalBreach();
+   return nullptr;
+}
+
+void* HarpoonKrakenTests::SendSmallChunks(void* arg) {
+   std::string address = *(reinterpret_cast<std::string*>(arg));
+   Kraken server;
+   server.MaxChunkSizeInBytes(2);
```

Comment:
OK, I see why you wanted to have this function. 
For now we want to have 10MB as the default so maybe 

1) Change file name to ```ChangeDefaultMaxChunkSizeInBytes```
2) (as a general rule I think that the name ```MaxChunkSizeInBytes``` would imply that it returns that value, not setting it. 

You could also have solved this by having the value being accessed inside the class by a function ``
```size_t MaxChunkSizeInBytes()``` (i.e. a getter, not a setter) and then in a MockKraken you overload this for whatever value you want to send.